src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -137,6 +137,51 @@ public void testStaticImportSeparated() throws Exception {
     }
 
     @Test
+    public void testNoGapBetweenStaticImports() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(ImportOrderCheck.class);
+        checkConfig.addAttribute("groups", "java, javax, org");
+        checkConfig.addAttribute("separated", "true");
+        checkConfig.addAttribute("ordered", "true");
+        checkConfig.addAttribute("option", "bottom");
+        final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
+
+        verify(checkConfig, getPath("InputImportOrderNoGapBetweenStaticImports.java"), expected);
+    }
+
+    @Test
+    public void testsortStaticImportsAlphabeticallyFalse() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(ImportOrderCheck.class);
+        checkConfig.addAttribute("groups", " java, javax, org");
+        checkConfig.addAttribute("separated", "true");
+        checkConfig.addAttribute("ordered", "true");
+        checkConfig.addAttribute("option", "top");
+        checkConfig.addAttribute("sortStaticImportsAlphabetically", "false");
+        final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
+
+        verify(checkConfig, getPath("InputImportOrdersortStaticImportsAlphabetically.java"),
+            expected);
+    }
+
+    @Test
+    public void testsortStaticImportsAlphabeticallyTrue() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(ImportOrderCheck.class);
+        checkConfig.addAttribute("groups", "java, javax, org");
+        checkConfig.addAttribute("separated", "true");
+        checkConfig.addAttribute("ordered", "true");
+        checkConfig.addAttribute("option", "top");
+        checkConfig.addAttribute("sortStaticImportsAlphabetically", "true");
+        final String[] expected = {
+            "4: " + getCheckMessage(MSG_ORDERING,
+                "javax.xml.transform.TransformerFactory.newInstance"),
+            "5: " + getCheckMessage(MSG_ORDERING, "java.lang.Math.cos"),
+            "6: " + getCheckMessage(MSG_ORDERING, "java.lang.Math.abs"),
+            };
```

Comment:
ident this line left 4 spaces. It should align with the `final`.

Reply:
Done.